### PR TITLE
Prepare for themed tokens - milestone

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #118, #119, #120, #122, #129

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

* Change token prefix from `auro` to `ds`
* Copy over token content and functionality from AuroDesignTokens to JetstreamDesignTokens
* Change `public` value from `true` to `false` for all deprecated tokens
* Update naming convention for dark mode tokens

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by jordanjones243** [comment]
Copying over token content and functionality to Jetstream repo can be verified by this PR: 
* https://github.com/Alaska-ITS/JetstreamDesignTokens/pull/1

File: scripts/config-dark.json [hunk 1]
```diff
@@ -136,9 +136,9 @@
         }
       ]
     },
-    "CSSOpacityCustomProp--darkerties": {
```
Comment (jason-capsule42) [inline scripts/config-dark.json:139]: What is this change used for?

File: src/breakpoint.json [hunk 2]
```diff
@@ -2,23 +2,23 @@
   "breakpoint": {
     "sm": {
       "value": "660px",
-      "public": true,
```
Comment (jason-capsule42) [inline src/breakpoint.json:5]: Why are we switching all these to false?

File: scripts/config-dark.json (hunk 1 above)
Comment (jordanjones243) [reply scripts/config-dark.json:139]: I made this change as the naming for this group was incorrect and not reflecting the name of the default tokens used for light theming. The name in the default tokens is `CSSOpacityCustomProperties` and this should be `CSSOpacityCustomProperties--dark`.

File: src/color-darkmode/alert.json [hunk 3]
```diff
@@ -2,50 +2,50 @@
...
         }
       },
       "warning": {
-        "onDark": {
+        "default": {
           "value": "{color.state.warning.500.value}",
           "public": true,
-          "onDark": true,
+          "default": true,
           "usage": "Warning color on light backgrounds",
```
[the remark this answers is not in the document]
Comment (jason-capsule42) [reply src/color-darkmode/alert.json:19]: "approved" after this change is made.

**Pull request merged**